malcolm/modules/pmac/velocityprofile.py

```diff
@@ -64,7 +65,8 @@ def __init__(
             tv2,  # type: float
             a,  # type: float
             v_max,  # type: float
-            settle_time=0  # type: float
+            settle_time=0,  # type: float
+            interval=0.002  # type: float
```

Comment:
Repeating 0.002 seems wrong. Should this be MIN_INTERVAL? Or should there be no default?

Reply:
I have changed the default to be 0 and for that to mean no quantization occurs.
Thus the default reverts to behaviour before the new parameter was added.